llvm/lib/Frontend/HLSL/RootSignatureMetadata.cpp

```diff
@@ -352,35 +374,34 @@ Error MetadataParser::parseDescriptorRange(mcdxbc::DescriptorTable &Table,
   else if (*ElementText == "Sampler")
     Range.RangeType = dxil::ResourceClass::Sampler;
   else
-    return make_error<GenericRSMetadataError>("Invalid Descriptor Range type.",
-                                              RangeDescriptorNode);
+    return makeRSError(formatv("Invalid Descriptor Range type. \n {0}",
```

Comment:
Presumably we don't want whitespace at the end of the line, and indenting the metadata node being printed by one seems like an odd choice. Double check these messages that print the MD node please.
```suggestion
    return makeRSError(formatv("Invalid Descriptor Range type.\n{0}",
```